src/components/questions/VideoSelector.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect } from 'react';
+import { StyleSheet, View, TouchableOpacity } from 'react-native';
+import { PERMISSIONS } from 'react-native-permissions';
+import { ImagePickerResponse, launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+import permissionCheck from '../utility/PermissionCheck';
+//@ts-ignore next-line
+import Video from 'react-native-video';
+import { FileSelectedData } from '../../models/questionOptionExtraData/FileSelectedData';
+import getLastItemFromSplit from '../../helpers/splitHelper';
+
+const VideoSelector = (props: {
+  value: string | undefined;
+  onVideoSelected: (videoPath: FileSelectedData | undefined) => void;
+}) => {
+  const [video, setVideo] = React.useState<string | undefined>('');
+  const [paused, setPaused] = React.useState<boolean>();
+  const [fullScreen, setFullScreen] = React.useState(false);
```

Comment:
Ik zag het bij de demo ook al. Ik heb liever dat het dan ook daadwerkelijk full screen wordt en niet gewoon vergroot op de pagina. Dus net zoals met YouTube ofzo gewoon een 'laag' over het scherm met daarin de video waarbij hij past in het scherm.